Add spaces to match_groups

KevinColemanInc opened the pull request.

FreedomBen commented:
@KevinColemanInc I noticed this PR has been open a really long time.  Did you ever hear back?  I wonder if this project has been abandoned.

KevinColemanInc commented:
I ended up just forking and writing my own :-/. The DIY approach let me customize the interface to better fit my business needs.

I assume this is abandoned. RIP.

FreedomBen commented:
Ah cool, did you open your own up?

KevinColemanInc commented:
Actually, it looks like the author accepted my other change in late-2019:

https://github.com/ofeldt/rubulex/commit/315bcfcf9b16ce956ac7502934589d5a05b6db29

Maybe they just didn't like this change.

KevinColemanInc commented:
My version was tied to the application's business logic. We were using regexes to capture information from pdf documents and websites. I added some nice features pre-loading the sample text and regex fields with the html and extracted pdf text, rendering the documents in an iframe, and opening up the regex groups into a new window for further parsing :)

KevinColemanInc commented:
If you need something quick, you could use https://rubular.com/

We had to self-host, because rubular changed the ruby version and broke all of our regexes that worked on 2.4 :-/

FreedomBen commented:
Oh haha, nice.  Yeah I've been using Rubular for years but it sends a POST back to it's server for each regex check.  Most of the time I don't care but I'm working on some sensitive code right now that I don't want sent to a closed-source backend.  I just dockerized Rubulex and deployed it for myself.  I offered the code back up, I guess we'll see if the maintainer wants it :man_shrugging:
